src/Phoenix.vue

```diff
@@ -1,14 +1,17 @@
 <template>
-  <v-app light id="Phoenix">
+  <v-app light id="Phoenix"
+         v-bind:class="{ loginGradient: showGradientBackground}"
```

Comment:
why do you use `v-bind:class` ?
I thought we could use vue's syntactical sugar & prefer the shorthand `:class`

Reply:
:man_shrugging: